commit 4042bec19dbb477d032a9b964da949aa952cc6d1
Author: Patrick Donnelly <pdonnell@ibm.com>
Date:   Thu Jan 11 14:46:32 2018 -0800

    mon: revert mds metadata argument name change
    
    Partial revert of PR #18512: d1877b68d90163897245a9801cff0ea419ff28e4
    
    Fixes: http://tracker.ceph.com/issues/22527
    
    Signed-off-by: Patrick Donnelly <pdonnell@redhat.com>

diff --git a/src/mgr/Mgr.cc b/src/mgr/Mgr.cc
index de0933da3d3..eed2bae7200 100644
--- a/src/mgr/Mgr.cc
+++ b/src/mgr/Mgr.cc
@@ -567,7 +567,7 @@ void Mgr::handle_fs_map(MFSMap* m)
       c->set_default("addr", stringify(info.addr));
 
       std::ostringstream cmd;
-      cmd << "{\"prefix\": \"mds metadata\", \"role\": \""
+      cmd << "{\"prefix\": \"mds metadata\", \"who\": \""
           << info.name << "\"}";
       monc->start_mon_command(
           {cmd.str()},
